src/main/ts/utils/EitherUtils.ts

```diff
@@ -11,3 +11,18 @@ export const rights = <R, A> (eithers: Array<Either<R, A>>): Array<A> => {
   }
   return r;
 };
+
+export const combine = <E, T> (eithers: Array<Either<E[], T>>): Either<E[], T[]> => {
+  let anyError = false;
+  let errors: E[] = [];
```

Comment:
Why isn't this `const` and using push?

Reply:
Because of line 23 where I concat an array of errors.